torch/csrc/jit/passes/batch_mm.cpp

```diff
@@ -142,10 +142,10 @@ struct TreeToken {
 
 void BatchMMBlock(Block* block) {
   enum class Side { LHS, RHS };
-  static const Symbol mm_kind = "mm"_sym;
-  static const Symbol add_kind = "add"_sym;
-  static const Symbol cat_kind = "cat"_sym;
-  static const Symbol dim_sym = "dim"_sym;
+  static const Symbol mm_kind = kmm;
+  static const Symbol add_kind = kadd;
+  static const Symbol cat_kind = kcat;
+  static const Symbol dim_sym = kdim;
   auto graph = block->owningGraph();
```

Comment:
Can you replace usages with constants?

Reply:
I'd prefer not to for now; `kmm` and co are going to get rewritten once namespacing gets added and replacing the usages makes my life harder later.

Reply:
Eh, fine...